projects/plugins/protect/src/js/routes/scan/index.jsx

```diff
@@ -163,28 +45,20 @@ const ScanPage = () => {
 		},
 	} );
 
-	// Render the appropriate section based on the scan status.
-	const renderSection = useMemo( () => {
-		if ( isScanInProgress( status ) ) {
-			return <ScanInProgressSection currentProgress={ status.currentProgress || 0 } />;
-		}
-
-		if ( status.error ) {
-			return <ErrorSection errorMessage={ status.errorMessage } errorCode={ status.errorCode } />;
-		}
-
-		return <ScanResultsSection />;
-	}, [ status ] );
-
 	return (
 		<OnboardingContext.Provider value={ onboardingSteps }>
 			<AdminPage>
-				<AdminSectionHero>
-					<ScanSectionNotices />
-					<Container horizontalSpacing={ 3 } horizontalGap={ 4 }>
-						{ renderSection }
-					</Container>
-				</AdminSectionHero>
+				<ScanAdminSectionHero />
+				{ ( ! status.error || ( status.error && numThreats > 0 ) ) && (
```

Comment:
```suggestion
				{ ( ! status.error || numThreats ) && (
```

Reply:
Nitpick, `status.error` will always be true on the right side of a `! status.error` check, so I don't think we don't need to check it :+1: